openshift/ocp_on_openstack.adoc

```diff
@@ -89,6 +89,123 @@ Push the `pass` changes to the git repository.
 git push
 ----
 
+
+== Post installation steps
+
+
+=== Install jump server
+
+To be able to SSH to any of the OCP cluster machines a _jump server_ is required.
+
+==== Requirements
+
+1. Obtain the name of the network interface created by the OCP installation process
+  * As a result of the OCP installation process on RHOS a network is created that allows communication between the cluster instances. 
+  * The name should resemble `ocp-<code>-openshift` where `ocp` is the prefix our ansible role applies to the installation and `<code>` is a dynamic string generated by the installation process.
+
+[NOTE]
+====
+For the sake of example the `ocp-sscpc` prefix will be used.
+====
+
+==== Create and configure the server
+
+[WARNING]
+====
+One _jump server_ will be required so the OCP cluster name prefix should be used when creating this VM.
```

Comment:
This verbiage is not clear. Are you saying that we need one jump_server / ocp network ?

Reply:
Yes, one jump server for each OCP cluster, as each cluster installation generates it's own network.

Reply:
Improved the docs to explain this.